src/main/java/racingcar/Position.java

```diff
@@ -0,0 +1,19 @@
+package racingcar;
+
+public class Position {
+    private static final int INITIAL_POSITION = 0;
+
+    private int position;
+
+    public Position() {
+        this.position = INITIAL_POSITION;
+    }
+
+    public void increase(int distance) {
+        this.position += distance;
+    }
+
+    public int toInt() {
```

Comment:
```suggestion
    public int value() {
```

변경을 위한 코멘트 아닙니다~
원시값포장이라면 `value`라고 생각해볼수있을것같네요 😄